src/main/kotlin/uk/gov/gdx/datashare/services/EventDataService.kt

```diff
@@ -133,9 +133,11 @@ class EventDataService(
     return EventNotification(
       eventId = event.id,
       eventType = subscription.eventType,
-      sourceId = event.dataId,
+      eventTime = event.eventTime,
+      ingestTime = event.whenCreated,
+      sourceId = if (includeData && EnrichmentField.SOURCE_ID in enrichmentFieldNames) event.dataId else null,
```

Comment:
I think this means we would need to have 2 sets of enrichment fields, one for the "thin" and one for the "thick". This is because someone like DWP will want the source ID on the "thin" call as well